mixer/adapter/stackdriver/helper/common_test.go

```diff
@@ -26,23 +28,33 @@ import (
 )
 
 func TestToOpts(t *testing.T) {
+	testSvcAcctFile, err := ioutil.TempFile("", "serviceAccount.json")
+	if err != nil {
+		t.Fatalf("cannot create a temp file for testing: %v", err)
+	}
+	svcAcctFilePath := testSvcAcctFile.Name()
+	defer os.Remove(svcAcctFilePath)
 	tests := []struct {
 		name string
 		cfg  *config.Params
 		out  []gapiopts.ClientOption // we only assert that the types match, so contents of the option don't matter
 	}{
 		{"empty", &config.Params{}, []gapiopts.ClientOption{}},
-		{"api key", &config.Params{Creds: &config.Params_ApiKey{}}, []gapiopts.ClientOption{gapiopts.WithAPIKey("")}},
+		{"api key", &config.Params{Creds: &config.Params_ApiKey{ApiKey: "test-api-key"}}, []gapiopts.ClientOption{gapiopts.WithAPIKey("test-api-key")}},
+		{"api key empty", &config.Params{Creds: &config.Params_ApiKey{}}, []gapiopts.ClientOption{}},
 		{"app creds", &config.Params{Creds: &config.Params_AppCredentials{}}, []gapiopts.ClientOption{}},
 		{"service account",
+			&config.Params{Creds: &config.Params_ServiceAccountPath{ServiceAccountPath: svcAcctFilePath}},
+			[]gapiopts.ClientOption{gapiopts.WithCredentialsFile(svcAcctFilePath)}},
+		{"service account empty",
```

Comment:
Should there be a case where service account file path is not empty but does not exist?

Reply:
Good call. Done.